go-controller/pkg/node/node.go

```diff
@@ -554,21 +572,7 @@ func configureSvcRouteViaBridge(bridge string) error {
 	if err != nil {
 		return fmt.Errorf("unable to get the gateway next hops, error: %v", err)
 	}
-	link, err := util.LinkSetUp(bridge)
-	if err != nil {
-		return fmt.Errorf("unable to get link for %s, error: %v", bridge, err)
-	}
-	for _, subnet := range config.Kubernetes.ServiceCIDRs {
-		gwIP, err := util.MatchIPFamily(utilnet.IsIPv6CIDR(subnet), gwIPs)
-		if err != nil {
-			return fmt.Errorf("unable to find gateway IP for subnet: %v, found IPs: %v", subnet, gwIPs)
-		}
-		err = util.LinkRoutesAdd(link, gwIP[0], []*net.IPNet{subnet}, config.Default.MTU)
-		if err != nil && !os.IsExist(err) {
-			return fmt.Errorf("unable to add route for service via shared gw bridge, error: %v", err)
-		}
-	}
-	return nil
+	return configureSvcRouteViaInterface(bridge, gwIPs)
```

Comment:
oh I see you addressed my earlier comment here, although it would still be nice to have the functions in the same file I guess since they are very closely related